vizro-core/docs/pages/user-guides/data.md

```diff
@@ -1,199 +1,20 @@
 # How to connect your dashboard to data
 
-This guide shows you how to connect your dashboard and your charts within the dashboard to data.
+Vizro supports two different types of data:
```

Comment:
@stichbury I tried to follow the new guidance on "How to" titles etc. but it was being written at the same time as I was doing these docs, so I might not have got it quite right - please do say.